.golangci.yaml

```diff
@@ -243,6 +243,11 @@ issues:
         - staticcheck
       text: "SA9003:"
 
+    # Exclude some deprecation errors
+    - linters:
+        - staticcheck
+      text: "SA1019:"
```

Comment:
```
controllers/servingruntime_validator.go:142:43: SA1019: sets.String is deprecated: use generic Set instead. new ways: s1 := Set[string]{} s2 := New[string]() (staticcheck)
func checkName(name string, internalNames sets.String, logStr string) error {
                                          ^
controllers/modelmesh/model_type_labels.go:26:11: SA1019: sets.String is deprecated: use generic Set instead. new ways: s1 := Set[string]{} s2 := New[string]() (staticcheck)
        mtLabels sets.String, pvLabels sets.String, rtLabel string) {
                 ^
controllers/modelmesh/model_type_labels.go:29:16: SA1019: sets.String is deprecated: use generic Set instead. new ways: s1 := Set[string]{} s2 := New[string]() (staticcheck)
        mtSet := make(sets.String, 2*len(rt.SupportedModelFormats))
                      ^

```

Reply:
I guess it would be good to make a follow-on issue with these kinds of things... since it would be better to adjust the code per the recommendation. For expediency though what you've done here looks good!

Reply:
Right, good point.

https://github.com/kserve/modelmesh-serving/issues/386